contents/tutorials/bounce-rate.md

```diff
@@ -0,0 +1,57 @@
+---
+title:  How to calculate bounce rate
+date: 2023-06-21
+author: ["ian-vanagas"]
+showTitle: true
+sidebar: Docs
+featuredImage: ../images/tutorials/banners/tutorial-15.png
+tags: ['insights', 'hogql']
+---
+
+Bounce rate is the percentage of users who leave your page immediately after visiting. It is a popular marketing metric showing the relevance and engagement of content for site visitors.  
+
+This tutorial shows you how to calculate bounce rate in PostHog. To get started, you need to install the [snippet](/docs/getting-started/install?tab=snippet) or [JavaScript SDK](/docs/libraries/js) and enable "record user sessions"  in [project settings](https://app.posthog.com/project/settings).
```

Comment:
Is there a way to do this without record user sessions?  Its our most expensive product and that costs add up quick

Perhaps using pageview and page leave events?